src/components/Example/Example.tsx

```diff
@@ -1,23 +1,170 @@
 import React from 'react'
 import { DeflyWalletConnect } from '@blockshake/defly-connect'
 import { DaffiWalletConnect } from '@daffiwallet/connect'
-import { WalletProvider, PROVIDER_ID, useInitializeProviders } from '../../index'
+import { WalletProvider, PROVIDER_ID, useInitializeProviders, Metadata, Network } from '../../index'
 import Account from './Account'
 import Connect from './Connect'
 import Transact from './Transact'
+import { CustomProvider } from '../../clients/custom/types'
+import algosdk from 'algosdk'
+import type _algosdk from 'algosdk'
+import { Buffer } from 'buffer'
+import { ICON as KMDICON } from '../../clients/kmd/constants'
 
 const getDynamicPeraWalletConnect = async () => {
   const PeraWalletConnect = (await import('@perawallet/connect')).PeraWalletConnect
   return PeraWalletConnect
 }
 
+class TestManualProvider implements CustomProvider {
```

Comment:
This is an example of a custom wallet client, in this case a really basic manual signing wallet that uses alert/prompt. Note: the clipbpard access is a bit hit and miss so I also console.log the value.

Reply:
Agreed, done (will push soon)